photoprism.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "PhotoPrism-Test",
+    "release": "12.4-RELEASE",
+    "artifact": "https://github.com/oridyn/iocage-plugin-photoprism",
+    "official": false,
+    "properties": {
+        "vnet": 1,
+        "dhcp": 1
```

Comment:
We ideally want to default to nat, is there some sort of requirement by the application for dhcp ?

Reply:
I've added `avahi-dameon` and `nginx` so that the plugin's UI can be accessed under `http://<jailname>.local` on port 80.
I prefer it to accessing through the IP and port.

Happy to change this though to NAT and port if you think that's better for these plugins.